docs/quickstart.md

```diff
@@ -12,6 +12,12 @@ To checkout the Marquez source code run:
 $ git clone git@github.com:MarquezProject/marquez.git && cd marquez
 ```
 
+#### INSTALL [DOCKER](https://github.com/MarquezProject/marquez/blob/master/Dockerfile)
+To install Docker run:
+```
+$ brew cask install docker
+```
```

Comment:
Depending on the local environment, `brew` my not be an option to install docker. Let's add a section (before the setup section) called `PREREQUISITES` instead linking to steps on how to install docker engine based on environment:

```
## Quickstart

#### PREREQUISITES

Before you begin, make sure you have installed:

* [Docker 17.05](https://docs.docker.com/install)+
* [Docker Compose](https://docs.docker.com/compose/install)

#### SETUP
.
.
```